contracts/LockedNORI.sol

```diff
@@ -223,8 +226,13 @@ contract LockedNORI is
     require(
       msg.sender == address(_bridgedPolygonNori),
       "lNORI: not BridgedPolygonNORI"
-    ); // todo verify this can only be invoked by the nori contract
-    // todo restrict such that only admin can invoke this function
+    );
+    require(
+      hasRole(TOKEN_GRANTER_ROLE, from) || hasRole(TOKEN_GRANTER_ROLE, operator),
```

Comment:
I'm surprised this works

Reply:
why?

Reply:
Mainly surprised because I thought I had already thought of everything :joy: 

Without the `require(msg.sender == address(_bridgedPolygonNori))` this would be problematic. But since that's restricted it should be fine :thinking: